upstairs/src/client.rs

```diff
@@ -389,11 +432,8 @@ impl DownstairsClient {
             match &mut out {
                 IOop::Write { dependencies, .. }
                 | IOop::WriteUnwritten { dependencies, .. }
-                | IOop::SerializedWrite { dependencies, .. }
-                | IOop::SerializedWriteUnwritten { dependencies, .. }
                 | IOop::Flush { dependencies, .. }
                 | IOop::Read { dependencies, .. }
-                | IOop::ExtentClose { dependencies, .. }
```

Comment:
I was a little confused here at first as to where `IOop::ExtentClose` had gone, but then
I understood I was looking at the upstairs version of `IOop` and not the downstairs version
(that github put just above this file).

I'm not in love with having two different enums having the same name and looking very
similar to each other, but since they live in different directories I'm not going to pitch a
fit about it.

Reply:
Yeah, I was on the fence about it, but since neither of them is public (any more), I decided to let it slide.  Calling them `UpstairsIOop` and `DownstairsIOop` would be very verbose...

Reply:
`UpIOop` and `DsIOop` could work too, but yeah, I'm okay with leaving it.